ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -74,6 +74,7 @@ internal constructor(
   val showLocationOfInterestSelectorRequests: @Hot Subject<ImmutableList<LocationOfInterest>> =
     PublishSubject.create()
 
+  // TODO: Cleanup this method
```

Comment:
Do we have an associated issue # we can put here?

Reply:
Will file a bug separately and update this.